icepyx/tests/test_variables.py

```diff
@@ -1,6 +1,27 @@
+import json
+
+import pytest
+
+from icepyx.core.variables import Variables
+from icepyx.core.query import Query
 import icepyx.core.variables as variables
 
 
+@pytest.fixture()
+def variables_obj_atl06():
+    return Variables(
+        "file", 
+        path='./test_data/processed_ATL06_20191130112041_09860505_006_01.h5',
```

Comment:
Hmm, we'll need to see if it's possible to avoid storing this 8MB HDF5 file in git, because it will slow down `git clone` for future developers. There's actually some work to prevent big files from being uploaded at #143, and we had some nasty work a few years ago to remove HDF5 files at #114.

There are some tests in the `test_behind_NSIDC_API_login.py` file which actually downloads data from NSIDC. Maybe it is possible to reuse some functionality from there?